@types/semver and path-browserify is normally included transitively by storybook which we delete, adding it here explicitely because it is needed for build

--- clients/package.json.orig	2025-05-05 18:48:27.441580106 +0600
+++ clients/package.json	2025-05-05 18:56:40.935703581 +0600
@@ -13,7 +13,6 @@
   },
   "homepage": "https://bitwarden.com",
   "scripts": {
-    "prepare": "husky",
     "lint": "eslint . --cache --cache-strategy content && prettier --check .",
     "lint:fix": "eslint . --cache --cache-strategy content --fix",
     "lint:clear": "rimraf .eslintcache",
@@ -27,8 +26,7 @@
     "docs:json": "compodoc -p ./tsconfig.json -e json -d . --disableRoutesGraph",
     "storybook": "ng run components:storybook",
     "build-storybook": "ng run components:build-storybook",
-    "build-storybook:ci": "ng run components:build-storybook --webpack-stats-json",
-    "postinstall": "patch-package"
+    "build-storybook:ci": "ng run components:build-storybook --webpack-stats-json"
   },
   "workspaces": [
     "apps/*",
@@ -37,58 +35,21 @@
   ],
   "devDependencies": {
     "@angular-devkit/build-angular": "18.2.12",
-    "@angular-eslint/schematics": "18.4.3",
-    "@angular/cli": "18.2.12",
     "@angular/compiler-cli": "18.2.13",
     "@babel/core": "7.24.9",
     "@babel/preset-env": "7.24.8",
-    "@compodoc/compodoc": "1.1.26",
-    "@electron/notarize": "2.5.0",
-    "@electron/rebuild": "3.7.1",
     "@lit-labs/signals": "0.1.2",
     "@ngtools/webpack": "18.2.12",
-    "@storybook/addon-a11y": "8.5.2",
-    "@storybook/addon-actions": "8.5.2",
-    "@storybook/addon-designs": "8.0.4",
-    "@storybook/addon-essentials": "8.5.2",
-    "@storybook/addon-interactions": "8.5.2",
-    "@storybook/addon-links": "8.5.2",
-    "@storybook/addon-themes": "8.5.2",
-    "@storybook/angular": "8.5.2",
-    "@storybook/manager-api": "8.5.2",
-    "@storybook/theming": "8.5.2",
-    "@storybook/web-components-webpack5": "8.5.2",
-    "@types/argon2-browser": "1.18.4",
-    "@types/chrome": "0.0.306",
-    "@types/firefox-webext-browser": "120.0.4",
-    "@types/inquirer": "8.2.10",
     "@types/jest": "29.5.12",
-    "@types/jquery": "3.5.32",
-    "@types/jsdom": "21.1.7",
-    "@types/koa": "2.15.0",
-    "@types/koa__multer": "2.0.7",
-    "@types/koa__router": "12.0.4",
-    "@types/koa-bodyparser": "4.3.7",
-    "@types/koa-json": "2.0.23",
-    "@types/lowdb": "1.0.15",
     "@types/lunr": "2.3.7",
     "@types/node": "22.10.7",
-    "@types/node-fetch": "2.6.4",
     "@types/node-forge": "1.3.11",
     "@types/papaparse": "5.3.15",
-    "@types/proper-lockfile": "4.1.4",
-    "@types/retry": "0.12.5",
+    "@types/semver": "^7.3.4",
     "@types/zxcvbn": "4.4.5",
-    "@typescript-eslint/rule-tester": "8.22.0",
-    "@typescript-eslint/utils": "8.22.0",
-    "@webcomponents/custom-elements": "1.6.0",
-    "@yao-pkg/pkg": "5.16.1",
     "angular-eslint": "18.4.3",
     "autoprefixer": "10.4.20",
     "babel-loader": "9.2.1",
-    "base64-loader": "1.0.0",
-    "browserslist": "4.23.2",
-    "chromatic": "11.25.2",
     "concurrently": "9.1.2",
     "copy-webpack-plugin": "12.0.2",
     "cross-env": "7.0.3",
@@ -96,55 +57,29 @@
     "electron": "34.0.0",
     "electron-builder": "24.13.3",
     "electron-log": "5.2.4",
-    "electron-reload": "2.0.0-alpha.1",
     "electron-store": "8.2.0",
     "electron-updater": "6.3.9",
-    "eslint": "8.57.1",
-    "eslint-config-prettier": "10.0.1",
-    "eslint-import-resolver-typescript": "3.7.0",
-    "eslint-plugin-import": "2.31.0",
-    "eslint-plugin-rxjs": "5.0.3",
-    "eslint-plugin-rxjs-angular": "2.0.1",
-    "eslint-plugin-storybook": "0.11.2",
-    "eslint-plugin-tailwindcss": "3.18.0",
     "html-loader": "5.1.0",
-    "html-webpack-injector": "1.1.4",
     "html-webpack-plugin": "5.6.3",
-    "husky": "9.1.7",
     "jest-diff": "29.7.0",
-    "jest-junit": "16.0.0",
     "jest-mock-extended": "3.0.7",
-    "jest-preset-angular": "14.1.1",
     "json5": "2.2.3",
-    "lint-staged": "15.4.1",
     "mini-css-extract-plugin": "2.9.2",
+    "path-browserify": "^1.0.1",
     "nx": "20.8.0",
     "postcss": "8.5.1",
     "postcss-loader": "8.1.1",
-    "prettier": "3.4.2",
-    "prettier-plugin-tailwindcss": "0.6.11",
-    "process": "0.11.10",
-    "remark-gfm": "4.0.0",
     "rimraf": "6.0.1",
     "sass": "1.83.4",
     "sass-loader": "16.0.4",
-    "storybook": "8.5.2",
-    "style-loader": "4.0.0",
-    "tailwindcss": "3.4.17",
-    "ts-jest": "29.2.2",
     "ts-loader": "9.5.2",
     "tsconfig-paths-webpack-plugin": "4.2.0",
-    "type-fest": "2.19.0",
     "typescript": "5.4.2",
     "typescript-eslint": "8.20.0",
     "typescript-strict-plugin": "2.4.4",
     "url": "0.11.4",
-    "util": "0.12.5",
-    "wait-on": "8.0.2",
     "webpack": "5.97.1",
-    "webpack-cli": "6.0.1",
-    "webpack-dev-server": "5.2.0",
-    "webpack-node-externals": "3.0.0"
+    "webpack-cli": "6.0.1"
   },
   "dependencies": {
     "@angular/animations": "18.2.13",
@@ -157,55 +92,28 @@
     "@angular/platform-browser-dynamic": "18.2.13",
     "@angular/router": "18.2.13",
     "@bitwarden/sdk-internal": "0.2.0-main.137",
-    "@electron/fuses": "1.8.0",
     "@emotion/css": "11.13.5",
-    "@koa/multer": "3.0.2",
-    "@koa/router": "13.1.0",
     "@microsoft/signalr": "8.0.7",
     "@microsoft/signalr-protocol-msgpack": "8.0.7",
     "@ng-select/ng-select": "13.9.1",
-    "argon2": "0.41.1",
-    "argon2-browser": "1.18.0",
     "big-integer": "1.6.52",
-    "bootstrap": "4.6.0",
-    "braintree-web-drop-in": "1.44.0",
     "buffer": "6.0.3",
-    "bufferutil": "4.0.9",
-    "chalk": "4.1.2",
     "commander": "11.1.0",
     "core-js": "3.40.0",
-    "form-data": "4.0.1",
-    "https-proxy-agent": "7.0.6",
-    "inquirer": "8.2.6",
-    "jquery": "3.7.1",
-    "jsdom": "26.1.0",
     "jszip": "3.10.1",
-    "koa": "2.16.1",
-    "koa-bodyparser": "4.4.1",
-    "koa-json": "2.0.2",
     "lit": "3.2.1",
-    "lowdb": "1.0.0",
     "lunr": "2.3.9",
-    "multer": "1.4.5-lts.1",
     "ngx-toastr": "19.0.0",
     "node-fetch": "2.6.12",
     "node-forge": "1.3.1",
     "oidc-client-ts": "2.4.1",
-    "open": "8.4.2",
     "papaparse": "5.5.2",
-    "patch-package": "8.0.0",
-    "popper.js": "1.16.1",
-    "proper-lockfile": "4.1.2",
-    "qrcode-parser": "2.1.3",
-    "qrious": "4.0.2",
     "rxjs": "7.8.1",
-    "tabbable": "6.2.0",
     "tldts": "6.1.77",
     "utf-8-validate": "6.0.5",
     "zone.js": "0.14.10",
     "zxcvbn": "4.4.2"
   },
   "overrides": {
-    "tailwindcss": "$tailwindcss",
     "@storybook/angular": {
       "zone.js": "$zone.js"
     },
--- i/apps/desktop/electron-builder.json
+++ w/apps/desktop/electron-builder.json
@@ -11,8 +11,6 @@
     "output": "dist",
     "app": "build"
   },
-  "afterSign": "scripts/after-sign.js",
-  "afterPack": "scripts/after-pack.js",
   "asarUnpack": ["**/*.node"],
   "files": [
     "**/*",
--- clients-desktop-v2024.10.0/apps/desktop/desktop_native/napi/package.json.orig	2024-10-17 19:25:31.222180130 +0200
+++ clients-desktop-v2024.10.0/apps/desktop/desktop_native/napi/package.json	2024-10-17 19:43:45.262219326 +0200
@@ -8,9 +8,6 @@
   },
   "author": "",
   "license": "GPL-3.0",
-  "devDependencies": {
-    "@napi-rs/cli": "2.18.4"
-  },
   "napi": {
     "name": "desktop_napi",
     "triples": {
